src/evo/deterministicmns.h

```diff
@@ -46,6 +46,12 @@ class CDeterministicMN
     static constexpr uint16_t CURRENT_MN_FORMAT = 0;
     static constexpr uint16_t MN_TYPE_FORMAT = 1;
 
+    uint256 proTxHash;
```

Comment:
Why they are still public and not `const`?
I don't mind if move it above (it's fine), but touching them and still keeping them public and non-const... It  doesn't look. Any code outside can easily change these data inside CDeterministicMN and make them inconsistent.

Reply:
@knst well no because we use `CDeterministicMNCPtr` (which is a `std::shared_ptr<const CDeterministicMN>`)